src/platform/silabs/wifi/wf200/WifiInterfaceImpl.cpp

```diff
@@ -1088,14 +1074,48 @@ static void wfx_events_task(void * p_arg)
     }
 }
 
-/****************************************************************************
- * @brief
- * Initialize the WF200 used by the two interfaces
- *****************************************************************************/
-static sl_status_t wfx_init(void)
+sl_status_t WifiInterfaceImpl::InitWf200Platform()
+{
+    sl_status_t status = SL_STATUS_OK;
+
+    VerifyOrReturnValue(!wifi_extra.Has(WifiInterface::WifiState::kStationStarted), SL_STATUS_OK);
+
+    ChipLogDetail(DeviceLayer, "STARTING WF200");
+
+    sl_wfx_host_gpio_init();
+
+    status = wfx_init();
+    VerifyOrReturnError(status == SL_STATUS_OK, status, ChipLogError(DeviceLayer, "WF200:init failed"));
```

Comment:
including the status in the error log will be helpful

Reply:
updated